RELEASE.md

```diff
@@ -0,0 +1,126 @@
+# Releasing PyTorch
+
+<!-- toc -->
+
+  - [General Overview](#general-overview)
+  - [Cutting release branches](#cutting-release-branches)
+    - [Making release branch specific changes](#making-release-branch-specific-changes)
+  - [Drafting RCs (Release Candidates)](#drafting-rcs-release-candidates)
+    - [Release Candidate Storage](#release-candidate-storage)
+    - [Cherry Picking Fixes](#cherry-picking-fixes)
+  - [Promoting RCs to Stable](#promoting-rcs-to-stable)
+- [Special Topics](#special-topics)
+  - [Updating submodules for a release](#updating-submodules-for-a-release)
+
+<!-- tocstop -->
+
+## General Overview
+
+Releasing a new version of PyTorch generally entails 3 major steps:
+
+1. Cutting a release branch and making release branch specific changes
+2. Drafting RCs (Release Candidates), and merging cherry picks
+3. Promoting RCs to stable
+
+## Cutting release branches
+
+Release branches are typically cut from the branch [`viable/strict`](https://github.com/pytorch/pytorch/tree/viable/strict) as to ensure that tests are passing on the release branch.
+
+Release branches *should* be prefixed like so:
+```
+release/{MAJOR}.{MINOR}
+```
+
+An example of this would look like:
+```
+release/1.8
+```
+
+Please make sure to create branch that pins divergent point of release branch from the main thunk, i.e. `orig/release/{MAJOR}.{MINOR}`
```

Comment:
What does this mean? From above it seems like the commands to run would be 

```bash
git fetch
git checkout viable/strict
git checkout -b release/1.9
git push origin
```

Reply:
Usually the way I'll do this (if we end up doing it later on in the process) is like so:
```
git checkout release/1.9
git checkout $(git merge-base HEAD master)
git checkout -b orig/release/1.9
git push origin orig/release/1.9
```